src/languages/es.js

```diff
@@ -0,0 +1,21 @@
+es: {
```

Comment:
Can you change the indentation in this file so that it matches the other language modules? That way, the built files in `lib` will get sane indentation too, thanks.